Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -58,6 +58,7 @@
 static const float s_kCGGradientOffsetPoint = 1E-45;
 
 enum _CGCoordinateMode : unsigned int { _kCGCoordinateModeDeviceSpace = 0, _kCGCoordinateModeUserSpace };
+enum _CGAntialiasToggle : unsigned int { _kCGToggleDefault = 2, _kCGToggleOff = 0, _kCGToggleOn = 1 };
```

Comment:
I understand the need for a trinary; I just don't think that we need to have a "yes/default/no" optionset for "Allow". Allow's default is On. If allow+default, do nothing.
If !allow+default, disable (effectively, set our default for "disallow" to be "no dwrite antialiasing" or w/e property)

---
In reply to: [100920282](https://github.com/Microsoft/WinObjC/pull/1938#discussion_r100920282) [](ancestors = 100920282)